dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -13,226 +13,412 @@ The Sliceviewer in Workbench has the joint functionality of the SpectrumViewer a
+    - The peaks should still be at integer HKL (rather than multiples of :math:`\sqrt{2}`)
 
-.. figure:: ../../../../docs/source/images/wb-sliceviewer51-roi.png
-   :class: screenshot
-   :width: 75%
-   :align: center
+6. Check gets the correct basis vectors for MDHisto workspaces
+##############################################################
 
-9. Save image
-#############
+This tests that the sliceviewer gets the correct basis vectors for an ``MDHisto`` object from a non-axis aligned cut.
 
-- Use the Save image toolbar button, in many instances, such as with peaks overlaid
-- In future there will also be a toolbar button to copy the image to clipboard
+1. Create the workspace
 
-10. Resizing
-###########
+.. code-block:: python
 
-- Play around with resizing the window and adjusting the size of the peak table**
+    ws = CreateMDWorkspace(Dimensions='3', Extents='-3,3,-3,3,-3,3',
+                       Names='H,K,L', Units='r.l.u.,r.l.u.,r.l.u.',
+                       Frames='HKL,HKL,HKL',
+                       SplitInto='2', SplitThreshold='10')
+    expt_info = CreateSampleWorkspace()
+    ws.addExperimentInfo(expt_info)
+    SetUB(ws, 1,1,2,90,90,120)
+    BinMD(InputWorkspace=ws, AxisAligned=False,
+        BasisVector0='[00L],r.l.u.,0,0,1',
+        BasisVector1='[HH0],r.l.u.,1,1,0',
+        BasisVector2='[-HH0],r.l.u.,-1,1,0',
+        OutputExtents='-4,4,-4,4,-0.25,0.25',
+        OutputBins='101,101,1', OutputWorkspace='ws_slice', NormalizeBasisVectors=False)
+
+2. Open ``ws_slice`` in the sliceviewer.
+
+    - The non-orthogonal view should be enabled (not greyed out).
 
-11. Alter the underlying workspace
-##################################
+3. Click the non-orthogonal view button
 
-- Delete the workspace and Sliceviewer should close
-- Rename the workspace and Sliceviewer should stay open and continue to work
-- Change the data in the workspace by cropping or running some algorithm (e.g. double the data ``SXD_MD_nonortho *= 2``)
-- Delete rows or re-integrate a PeaksWorkspace that is overlaid.
+    - Rectangular gridlines should appear (as in this case 110 is orthogonal to 001).
 
+7. Check non-orthogonal view is disabled for non-Q axes
+#######################################################
+Check that the non-orthogonal view is disabled for non-Q axes such as energy
 
-.. |PickTabAddPeakButton.png| image:: ../../../../docs/source/images/PickTabAddPeakButton.png
+1. Create a workspace with energy as the first axis.
+
+.. code-block:: python
+
+    ws_4D = CreateMDWorkspace(Dimensions=4, Extents=[-1, 1, -1, 1, -1, 1, -1, 1], Names="E,H,K,L",
+                                  Frames='General Frame,HKL,HKL,HKL', Units='meV,r.l.u.,r.l.u.,r.l.u.')
+    expt_info_4D = CreateSampleWorkspace()
+    ws_4D.addExperimentInfo(expt_info_4D)
+    SetUB(ws_4D, 1, 1, 2, 90, 90, 120)
 
+2. Confirm that when the Energy axis is viewed (as X or Y) the non-orthogonal view is disabled.
```

Comment:
```suggestion
2. Open `` ws_4D`` in sliceviewer. Confirm that when the Energy axis is viewed (as X or Y) the non-orthogonal view is disabled.
```
From the code snippet it's obvious but testers probably have lots in the ADS at this point!